infor-design/enterprise
# 7603 - Datepicker Date Range Render

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pull request will fix a bug in datepicker range not rendering properly in modal IOS.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/7603

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/datepicker/test-modal-range.html
- Test this in `IOS Safari`
- Click `Show Modal`
- Go to the last Datepicker Range
- Click the `icon` to show the popup
- See the selected range, rendered properly

**Included in this Pull Request**:
